src/benchmarks/perf/cpu/src/main.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright(c) The Maintainers of Nanvix.
+// Licensed by the MIT License.
+
+#![no_std]
+#![no_main]
+
+//==================================================================================================
+// Imports
+//==================================================================================================
+
+extern crate alloc;
+
+use ::alloc::vec;
+use ::alloc::vec::Vec;
+use ::micromath::F32Ext;
+
+use ::core::alloc::{
+    GlobalAlloc,
+    Layout,
+};
+
+use ::nvx::sys::error::Error;
+
+use ::serde::Deserialize;
+use ::serde_json::de::from_str;
+
+use ::fastrand::Rng;
```

Comment:
Remove all spaces in between imports to follow the same standard used across the project.